conf/test_cram.config

```diff
@@ -0,0 +1,25 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run qbic-pipelines/bamtofastq -profile test_cram
+ */
+
+
+params {
+  config_profile_name = 'Test profile'
+  config_profile_description = 'Minimal test dataset to check pipeline function'
+  // Limit resources so that this can run on Travis
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 48.h
+
+  cram_files = true
+  input = [
+          'https://raw.githubusercontent.com/BrunoGrandePhD/bamtofastq/master/testdata/First_SmallTest_Paired.cram',
+          'https://raw.githubusercontent.com/BrunoGrandePhD/bamtofastq/master/testdata/Second_SmallTest_Paired.cram'
```

Comment:
This could be replaced with URLs from the main repo immediately prior to merging.

Reply:
sounds good, this is how we usually have done it. Bit of a chicken and egg problem.